packages/fxa-settings/src/components/Settings/ProductPromo/index.test.tsx

```diff
@@ -63,7 +63,7 @@ describe('ProductPromo', () => {
 
     const { container } = renderWithLocalizationProvider(
       <AppContext.Provider value={mockAppContext({ account })}>
-        <ProductPromo />
+        <ProductPromo type={ProductPromoType.Settings} />
```

Comment:
Not too familiar with the ProductPromo component, so just to double check: even though it's named ProductPromo, it only promotes Monitor Plus correct?  That's the only proudct?

Reply:
That is currently the only product, correct. When we first talked about this promo we talked about reusing it for other products, hence the more generic name, but it's only set up now for Monitor. Later if/when we use it we'll probably pass a `product` prop or something.